internal/mode/static/manager.go

```diff
@@ -399,6 +405,33 @@ func registerControllers(
 	return nil
 }
 
+func createTelemetryJob(
+	cfg config.Config,
+	dataCollector telemetry.DataCollector,
+	nginxChecker *nginxConfiguredOnStartChecker,
```

Comment:
Probably simpler to pass the ready channel instead of the entire nginxChecker here.